.github/workflows/behaviour.yml

```diff
@@ -71,10 +71,23 @@ jobs:
       - name: Change MySQL authentication method
         run: mysql -h127.0.0.1 -uroot -ppassword -e "ALTER USER 'root'@'%' IDENTIFIED WITH mysql_native_password BY 'password'; FLUSH PRIVILEGES;"
 
+      - name: Install maildev
+        run: |
+          docker run -d -p 1080:1080 -p 1025:1025 maildev/maildev
```

Comment:
Maildev looks bad.. 0 commit since 10 months.
Why not use https://github.com/axllent/mailpit instead? (Mailpit is the unofficial mailhog successor and have the same amount of star than maildev)

Reply:
QA teams use maildev in their environment so I prefer to stick to their choice for now, if we decide to change this it should be discussed with them and changed everywhere I think
So far it does the job anyway